### DISCUSSION TITLE
How to copy a custom node's text in TipTap and paste in an external program

### DISCUSSION THREAD

**Discussion by zawilliams** [ueberdosis/tiptap#857] [General Questions] [answered]
Take a look at this example: https://tiptap.dev/suggestions

If you select and copy the second paragraph ("This is an example how to mention...") and then paste it anywhere that isn't TipTap, it will copy the following: 

"This is an example how to mention some users like  or . Try to type @ and a popup (rendered with tippy.js) will appear. You can navigate with arrow keys through a list of suggestions."

Notice it doesn't copy `@Philip Kuhn` or `@Hans Pagel`.

Is there a value property or anything I'm missing with making sure a custom node can have a text value that can be copied? Or is this a ProseMirror issue?

**Comment by hanspagel** [comment, marked as answer]
Thanks for reporting! This is a bug, so I created a new issue based on your report:
https://github.com/ueberdosis/tiptap/issues/858

**Reply by zawilliams** [reply]
No prob! Thanks!
